db/index.js

```diff
@@ -0,0 +1,27 @@
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/productSelection');
+
+var itemSchema = mongoose.Schema({
+  _id: Number,
+  price: String,
+  totalRatings: Number,
+  averageRating: Number,
+  totalQuestions: Number,
+  colors: [String],
+  sizes: [String],
+  quantityCanBuy: Number
+});
+
+var Item = mongoose.model('Item', itemSchema);
+
+var getItemById = function (id, callback) {
+  Item.findById(id, callback);
+}
```

Comment:
imo it'd be better to eventually refactor to use exclusively promises (so as to avoid mixing promises w/ callbacks)

recommended: 
```
const getItemById = (id) => Item.findById(id);
```

aka

```
const getItemById = function (id) {
  return Item.findById(id);
}
```